README.md

```diff
@@ -21,40 +21,37 @@ To run AI applications, your system needs
   - To run AI applications (test machine): RyzenAI processor
   - To build this repository (build machine): Any x86 processors, but recommend AMD processor :wink:
 * Operating System: Ubuntu >= 22.04
-* Linux Kernel: v6.8 with IOMMU SVA support (see below)
+* Linux Kernel: v6.10 or above. (See [Linux compilation and installation](#linux-compilation-and-installation)
+  - Due to Linux API change, XDNA driver doesn't always keep supporting old version.
 * Installed XRT base package (or you can install it along the
   following recipe)
   - To make sure the XRT base package works with the plug-in package, better build it from `xrt` submodule in this repo (`<root-of-source-tree>/xrt`)
   - Refer to https://github.com/Xilinx/XRT for more detailed information.
 
-*Important*: IOMMU SVA in Linux kernel support is required.
-
 ## Linux compilation and installation
 
-You need to manually build 6.8 Linux kernel packages by following below steps.
+Since Linux v6.10 offically supports AMD IOMMU SVA, we can play with original Linux source code.
```

Comment:
... we can work with upstream Linux kernel source.

Reply:
fixed